cypress/support/pages/homeСatalogue.pageObject.js

```diff
@@ -17,6 +17,17 @@ class HomeAndCataloguePageObject extends PageObject {
     cy.contains('.hrefch', product)
       .click();
   }
+
+  addToCartBtn() {
```

Comment:
Function name 'addToCartBtn' should be a verb or verb phrase. Consider renaming it to 'clickAddToCartBtn' to better reflect its action.